src/04-modality-specific-files/10-eye-tracking.md

```diff
@@ -0,0 +1,187 @@
+
+The OPTIONAL [`acq-<label>`](../99-appendices/09-entities.md#acq) refers to a custom label the user MAY use to distinguish a different set of parameters used for acquiring the same modality. Acquisition labels corresponds mostly to imaging records (for example eye-tracking with fMRI) but can also be specified when combining eye-tracking and imaging methods.
+
+If more than one run of the same task and acquisition are acquired during the same session, the [`run-<index>`](../99-appendices/09-entities.md#run) entity MUST be used: _run-1, _run-2, _run-3, and so on. If only one run was acquired the run-<index> can be omitted.
+
+### Sidecar JSON document (`*_eyetrack.json`)
+
+Generic fields MUST be present:
+
+{{ MACROS___make_metadata_table(
+   {
+      "TaskName": ("REQUIRED", "A RECOMMENDED convention is to name resting state task using labels beginning with `rest`."),
+   }
+) }}
+
+SHOULD be present: 
+
+For consistency between studies and institutions, we encourage users to extract the values of these fields from the actual raw data. Whenever possible, please avoid using ad hoc wording.
+
+{{ MACROS___make_metadata_table(
+   {
+      "InstitutionName": "RECOMMENDED",
+      "InstitutionAddress": "RECOMMENDED",
+      "Manufacturer": (
+         "RECOMMENDED",
+         "Manufacturer of the eye-tracking system (for example "
+         '`"SR-Research"`, `"Tobii"`, `"SMI"`, `"Gazepoint"`, '
+         '`"Pupil Labs"`, `"Custom built"`, `"Other"`) '
+      ),
+      "ManufacturersModelName": (
+         "RECOMMENDED",
+         "Manufacturer's designation of the eye-tracker model (for example"
+         '`"Eye-link 1000"`).'
+         ),
+      "SoftwareVersion": "RECOMMENDED",
+      "TaskDescription": "RECOMMENDED",
+      "Instructions": (
+         "RECOMMENDED", 
+         "Text of the instructions given to participants before the experiment."),
+      "CogAtlasID": "RECOMMENDED",
+      "CogPOID": "RECOMMENDED",
+      "DeviceSerialNumber": "RECOMMENDED",
+   }
+) }}
+
+
+Specific ET fields MUST be present:
+
+{{ MACROS___make_metadata_table(
+   {
+      "SamplingFrequency_eyetrack": "REQUIRED",
+      "SampleCoordinateUnit": "REQUIRED",
+      "SampleCoordinateSystem": "REQUIRED",
+      "EnvironmentCoordinates": "REQUIRED",
+      "ScreenSize": "REQUIRED",
+      "ScreenResolution": "REQUIRED",
+      "ScreenDistance": (
+         "REQUIRED",
+         "Screen distance in cm (e.g 60 for a screen distance of 60 cm), "
+         'if no screen use "n/a". '
+         "For MRI, it corresponds to the distance between the head-coil mirror "
+         "to the projection screen for example."),
```

Comment:
Those 3 should probably go in the events.json file to maximize the reuse of those metadata even no eyetracking data is present:

see

https://bids-specification.readthedocs.io/en/latest/04-modality-specific-files/05-task-events.html#stimulus-presentation-details